soroban-env-host/Cargo.toml

```diff
@@ -79,6 +79,7 @@ next = ["soroban-env-common/next"]
 tracy = ["dep:tracy-client", "soroban-env-common/tracy"]
 recording_auth = []
 bench = []
+time = []
```

Comment:
I think we could gate the tracking of time functionality on the target instead of a feature. I would favor this for a couple reasons:
- Every new feature we add increases build times and testing. We don't need to avoid adding features. But we should be intentional about adding them.
- Gating the functionality on the wasm32 family would more clearly communicate what use case the code is gated out of. I doubt there will be other situations we need to disable time.

Reply:
That's actually a great idea, thanks for the suggestion. I agree we should the target gate instead.

Reply:
Yes agreed this is better!

Reply:
Yeah definitely makes sense

Reply:
@leighmcculloch changed this in the latest commit.